### ISSUE TITLE
vcocalc.py should have an option to use the default REFDIV of 2

### ISSUE THREAD

**Issue by kilograham** [raspberrypi/pico-sdk#1451]
#952 added support for non-standard REFDIV values in vcocalc.py (though doesn't add command line arguments for them)

REFDIV 1 is now supported, so will be used in preference to REFDIV 2 in many cases (and set_sys_clock_pll by default only uses the default REFDIV value of 2)

perhaps adding the refdiv-min and refdiv-max aeguments and defaulting the former to 2 would be a reasonable way to solve the problem

**Comment by kilograham** [comment]
cc @alastairpatrick

Note, REFDIV is only settable by `#define` which was intentional as it is common, perhaps we should add some blurb to the vcocalc.py output to say how to set up that clock

**Comment by kilograham** [comment]
note actually, maybe we should default both refdivs min/max to 2, or default them to 1 and 63 as they are now, but add an flag to allow you to pick a REFDIV other than 2

**Comment by alastairpatrick** [comment]
Regarding "REFDIV is only settable by #define which was intentional as it is common," is that actually true? I haven't tried but the RP2040 datasheet suggests that the SYS and USB PLLs have independent REFDIVs, so if that's actually true, it might make sense to have separate REFDIV #defines for each PLL.

Alternatively, the USB PLL will basically always want to run at 48MHz. 48MHz can be achieved with both REFDIV=1 and REFDIV=2:
```
Requested: 48.0 MHz
Achieved: 48.0 MHz
REFDIV: 1
FBDIV: 120 (VCO = 1440.0 MHz)
PD1: 6
PD2: 5
```
So it should be possible to make things work with just a single common REFDIV #define, set to either REFDIV=1 or REFDIV=2, have USB clock domain always operate at 48MHz and the SYS clock domain operate at whatever the desired frequency was passed to vcocalc.py.

**Comment by lurch** [comment]
> it might make sense to have separate REFDIV #defines for each PLL.

This was (kind of) touched upon recently in #1272

**Comment by will-v-pi** [comment]
Fixed with SDK 2.0.0
